test/Interpreter/SR-10600.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
```

Comment:
I have an executable test case in https://github.com/apple/swift/pull/24759/files as well. This is interesting, because what we really want to test here is this program built and run against the old runtime---not the one we built! Do we have a way to force that? Otherwise, we're not testing the back-deployment that we're intending to test.

Reply:
Rather than do something specific to this test, I think it’d make more sense for CI to run tests in a mode where all execution tests run against the runtime in the OS. That would help us catch other backward deployment issues, as well as check that the compatibility patches don’t regress other behavior.

Reply:
Yeah, I agree.